src/acceleration/config/AccelerationConfiguration.cpp

```diff
@@ -399,7 +414,8 @@ void AccelerationConfiguration::addTypeSpecificSubtags(
     tagData.addAttribute(attrMesh);
     tag.addSubtag(tagData);
   } else if (tag.getName() == VALUE_IQNILS) {
-    XMLTag tagInitRelax(*this, TAG_INIT_RELAX, XMLTag::OCCUR_ONCE);
+
+    XMLTag tagInitRelax(*this, TAG_INIT_RELAX, XMLTag::OCCUR_NOT_OR_ONCE);
     tagInitRelax.setDocumentation("Initial relaxation factor.");
```

Comment:
I think we should add information on the default behavior in all of these documentation strings.

```suggestion
    tagInitRelax.setDocumentation("Initial relaxation factor. If tag is not provided, an initial relaxation of 0.1 is used.");
```

similarly below; also for filter and preconditioner